R/get_phylopic.R

```diff
@@ -31,36 +36,50 @@ get_phylopic <- function(uuid = NULL, format = "vector") {
   if (!is.character(uuid)) {
     stop("`uuid` is not of class character.")
   }
+  if (as.character(format) %in% c("512", "1024", "1536")) {
```

Comment:
Nice! It's a good idea that we start including lifecycle warnings (here and palaeoverse)